spec/code_climate/test_reporter/git_spec.rb

```diff
@@ -79,5 +79,19 @@ def self.root
       end
     end
 
+    describe 'committed_at_from_ci_or_git' do
+      it 'returns the committed_at from ci' do
+        allow(Ci).to receive(:service_data).and_return({committed_at: 'ci-committed_at'})
```

Comment:
instead of `ci-committed_at`, WDYT about putting a value such as `"1484768698"` (result of `Time.now.to_i.to_s`), which is what we expect codeship to set in this environment variable